lib/versioning/node/index.spec.ts

```diff
@@ -10,22 +10,35 @@ describe('versioning/node/index', () => {
     DateTime.local = dtLocal;
   });
 
-  test.each`
-    currentValue | rangeStrategy | currentVersion | newVersion  | expected
-    ${'1.0.0'}   | ${'replace'}  | ${'1.0.0'}     | ${'v1.1.0'} | ${'1.1.0'}
-    ${'~8.0.0'}  | ${'replace'}  | ${'8.0.2'}     | ${'v8.2.0'} | ${'~8.2.0'}
-  `(
-    'getNewValue($currentValue, $rangeStrategy, $currentVersion, $newVersion, $expected) === $expected',
-    ({ currentValue, rangeStrategy, currentVersion, newVersion, expected }) => {
-      const res = nodever.getNewValue({
+  describe('getNewValue', () => {
+    test.each`
+      currentValue | rangeStrategy | currentVersion | newVersion   | expected
+      ${'1.0.0'}   | ${'replace'}  | ${'1.0.0'}     | ${'v1.1.0'}  | ${'1.1.0'}
+      ${'~8.0.0'}  | ${'replace'}  | ${'8.0.2'}     | ${'v8.2.0'}  | ${'~8.2.0'}
+      ${'erbium'}  | ${'replace'}  | ${'12.0.0'}    | ${'v14.1.4'} | ${'fermium'}
+      ${'Fermium'} | ${'replace'}  | ${'14.0.0'}    | ${'v16.1.6'} | ${'gallium'}
+      ${'gallium'} | ${'pin'}      | ${'16.0.0'}    | ${'v16.1.6'} | ${'16.1.6'}
+      ${'gallium'} | ${'bump'}     | ${'16.0.0'}    | ${'v16.1.6'} | ${'gallium'}
+      ${'gallium'} | ${'auto'}     | ${'16.0.0'}    | ${'v16.1.6'} | ${'gallium'}
```

Comment:
Please double-check if these test cases assert the right things, I was not entirely sure how to handle all the different range strategies.